README.md

```diff
@@ -55,13 +56,16 @@ Tested with the following:
 
 #### Optional
 
-* `letsencrypt_certbot_args` - Additional command line args to Certbot.
+* `letsencrypt_certbot_args` - Additional command line args to be passed to Certbot-- will be combined with `letsencrypt_certbot_default_args`. See [the Certbot docs](https://certbot.eff.org/docs/using.html) for arguments you may pass.
+* `letsencrypt_certbot_default_args` - Default command line args passed to certbot
```

Comment:
I'd add here to the end of the line `Please see "defaults/main.yml" what the default arguments are.`. Also, you could add `To override all the arguments to Certbot, for example to use another plugin, set them using this variable.`. This makes the comment on line 67 redundant.